ENGLISH.md

```diff
@@ -95,11 +95,12 @@ sudo apt install <packagename>
         3. Elevate user privilege: `alter user "ubuntu" with superuser;`
     3. Now we will set the database up with `bundle exec rake db:setup`
     4. Run all migration files by `bundle exec rake db:migrate`
-8. The initial setup to run the server has been done. To run the server in test environment, you can run 
+8. Create `.env` file in the root directory and copy over the contents from `.env.default` to `.env`
```

Comment:
Should also note that the env vars in `.env` need to be populated with any values that need to be filled in

Reply:
Done